includes/classes/PullListTable.php

```diff
@@ -258,7 +259,7 @@ public function column_date( $post ) {
 	public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 			case 'name':
-				return $item['post_title'];
+				return $item->post_title;
 			case 'url':
```

Comment:
I know this is unrelated to any changes here but also doesn't seem we ever use a `url` column and if so, probably worth removing this case statement here as we're modifying other things in this UI

Reply:
I agree that even some light cleanup is worthwhile while we're working on this section of the code.

Reply:
@dkotter Updated.